### PR TITLE
Progress bar color fix

### PR DESCRIPTION
**Pull request by aganapuram**
### Summary
A small PR for for fixing the color prop issue. 
Resolves issue: https://github.com/cerner/terra-core/issues/370

### Additional Details
If you have anything else that you think may be relevant to this issue, list it here. Additional information can help us better understand your changes and speed up the review process.

Please add your name to the [CONTRIBUTORS.md] file. Adding your name to the [CONTRIBUTORS.md] file signifies agreement to all rights and reservations provided by the [License].

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-progress-bar/tests/jest/ProgressBar.test.jsx [hunk 1]
```diff
@@ -8,32 +8,32 @@ it('should render a default component', () => {
 });
 
 it('should render a ProgressBar component with tiny heightSize and 15% fill', () => {
-  const wrapper = shallow(<ProgressBar heightSize="tiny" value={15} />);
+  const wrapper = shallow(<ProgressBar heightSize="tiny" value={15} color="#8ccc62" />);
```
Comment (Matt-Butler) [inline packages/terra-progress-bar/tests/jest/ProgressBar.test.jsx]: Can you leave a couple examples where color is not provided?

Comment (aganapuram) [inline packages/terra-progress-bar/tests/jest/ProgressBar.test.jsx]: yes, I will.

File: packages/terra-progress-bar/docs/README.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ import ProgressBar from 'terra-progress-bar';
   heightSize="tiny"
   value={3}
   max={10}
+  color="Orange"
```
Comment (bjankord) [inline packages/terra-progress-bar/docs/README.md]: nit: change this to a hexidecimal value, e.g. `#06dce8` rather than a color keyword.

Comment (aganapuram) [inline packages/terra-progress-bar/docs/README.md]: yes, I will

**Comment by alex-bezek** [comment]
+1

**Pull request merged**